packages/react-admin/src/posts.jsx

```diff
@@ -0,0 +1,72 @@
+// in src/posts.js
+import * as React from "react";
```

Comment:
Shouldn't this be automatically created by react admin?

Reply:
> Shouldn't this be automatically created by react admin?

Yes, more or less...

At this moment I'm at the 2nd steps of 3, so I will working on it later: 
1) add (statically) react example app
2) add (statically) react-admin example app
3) add (dinamically) react-admin app (I will build it according to the db resourses exposes by platformatic db)